### PR TITLE
docs: add a troubleshooting page

### LINKED ISSUE #1098: Troubleshooting page in docs
### Description

Add a troubleshooting page that explains :
- how to solve `[TMDB] failed to retrieve XXX` because TMDB is blocked by the DNS server
- how to fix some slowdowns / ipv6 errors with the `forceIpv4First` env var

### Desired Behavior

/

### Additional Context

_No response_

### Code of Conduct

- [x] I agree to follow Jellyseerr's Code of Conduct

### PR DESCRIPTION
**Pull request by gauthier-th**
#### Description

This troubleshooting page explains how to solve dns blockage issues from ISPs and how to fix some slowdowns / ipv6 errors with the forceIpv4First environment variable.

#### To-Dos

- [ ] Successful build `pnpm build`
- [ ] Translation keys `pnpm i18n:extract`
- [ ] Database migration (if required)

#### Issues Fixed or Closed

- Fixes #1098

### CODE REVIEW

File: docs/troubleshooting.mdx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+  <TabItem value="docker-compose" label="Docker Compose">
+
+Add the following to your `compose.yaml`:
+```yaml
+---
+services:
+  jellyseerr:
+    environment:
+      - FORCE_IPV4_FIRST=true
```
Comment (fallenbagel) [inline docs/troubleshooting.mdx:104]: It's `forceIpv4First`

**Review by Fallenbagel — CHANGES_REQUESTED** [review]

File: docs/troubleshooting.mdx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+Sometimes there are configuration issues with IPV6 that prevent the hostname resolution from working correctly.
+
+You can try to force the resolution to use IPV4 first by setting the `FORCE_IPV4_FIRST` environment variable to `true`:
+
+<Tabs groupId="methods" queryString>
+  <TabItem value="docker-cli" label="Docker CLI">
+
+Add the following to your `docker run` command:
+```bash
+-e "FORCE_IPV4_FIRST=true"
```
Comment (fallenbagel) [inline docs/troubleshooting.mdx:91]: It's `forceIpv4First`

**Review by Fallenbagel — CHANGES_REQUESTED** [review]

File: docs/troubleshooting.mdx (hunk 1 above)
Comment (gauthier-th) [reply docs/troubleshooting.mdx:104]: Both are ok: https://github.com/Fallenbagel/jellyseerr/blob/develop/next.config.js#L7

Comment (fallenbagel) [reply docs/troubleshooting.mdx:104]: Oh yeah lmfao

File: docs/troubleshooting.mdx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## [TMDB] failed to retrieve/fetch XXX
+
+### Option 1: Change your DNS servers
+
+This error usually comes from your DNS servers not being able to resolve the hostname of the TMDB API, because your Internet Service Provider (ISP) is blocking it.
```
Comment (fallenbagel) [inline docs/troubleshooting.mdx]: ISP blocking could be both by just simple dns or full block. Please add details on dns method and full block seperately where full block requires http proxy

**Review by Fallenbagel — CHANGES_REQUESTED** [review]

**Review by Fallenbagel — APPROVED** [review]
Lgtm

**Pull request merged**